src/odemis/acq/fastem.py

```diff
@@ -587,4 +590,11 @@ def _pass_future_progress(sub_f, start, end):
 
     if len(das) != 1:
         logging.warning("Expected 1 DataArray, but got %d: %r", len(das), das)
+
+    # Switch immersion mode back on, so we can do use the system normally.
+    stream.emitter.immersion.value = True
+
+    # FIXME auto blanking not working properly, so force beam blanking after image acquisition for now.
```

Comment:
Why did you add this line? I thought the autoblanking works now.

Reply:
Wilco just let me know that for some reason it doesn't always work

Reply:
ok